README.md

```diff
@@ -1,2 +1,110 @@
-# recipe_app_rails
-Good repo
+# Blog App
+
+> The recipe is created with ruby on rails and allows logged in users to create recipes from food items they have available. It also allows them to view public recipes shared by other users.
+
+## Project screenshots
+
+![screenshot()]()
+
+Additional description about the project and its features.
+
+## Built With
+
+- RUBY on RAILS
+- PostgreSQL
+
+## How to Setup
```

Comment:
- [ ] I suggest to update `How to setup` section for example, using `git clone <repo link>`, `cd <repo name>` commands and their descriptions as you did in the `Install gems` section. Please make it simple and clear.